src/Symfony/Bundle/FrameworkBundle/Validator/ConstraintValidatorFactory.php

```diff
@@ -42,8 +42,8 @@
  */
 class ConstraintValidatorFactory implements ConstraintValidatorFactoryInterface
 {
-    protected $container;
-    protected $validators;
+    private $container;
+    private $validators;
```

Comment:
The class is marked as `@final` since 3.3, which should trigger the notice. The upgrade files and changelogs also mention those properties will be removed.